frontends/web/src/routes/buy/moonpay.tsx

```diff
@@ -83,11 +84,13 @@ class Moonpay extends Component<Props, State> {
         if (!account) {
             return null;
         }
-        const name = code === 'btc' || code === 'tbtc' ? 'Bitcoin' : 'crypto';
+        const name = (code && isBitcoin(code)) ? 'Bitcoin' : 'crypto';
         return (
             <div class="contentWithGuide">
                 <div class="container">
-                    <Header />
+                    <div class={style.header}>
+                        <Header title={<h2>{t('buy.info.title', { name })}</h2>} />
```

Comment:
Alternatively, spell the crypto name for all coins? Buy Litecoin, Buy Ethereum, ...